src/components/common/Extensions/ExtensionStatusBadge/ExtensionStatusBadge.tsx

```diff
@@ -50,28 +50,29 @@ const ExtensionStatusBadge = ({
 
   if (!isInstalledExtensionData(extensionData)) {
     status = MSG.notInstalled;
-  } else if (!extensionData.isInitialized) {
-    status = MSG.notEnabled;
```

Comment:
Just checking you intended to remove the "Disabled" tag? I believe it's supposed to represent the state after installation but before it's been enabled. 

If so, the message can be removed from the MSG object above.

Reply:
Good spot. Installed and disabled feel very similar but it might make more sense to make disabled the default status